controllers/default.py

```diff
@@ -404,19 +409,25 @@ def contests():
     contests = []
     cal = pdt.Calendar()
 
-    table = TABLE(_class="centered striped")
+    button_class = "btn-floating btn-small accent-4 tooltipped"
+    view_link_class = button_class + " green view-contest"
+    reminder_class = button_class + " orange set-reminder"
+
+    table = TABLE(_class="centered striped", _id="contests-table")
     thead = THEAD(TR(TH("Contest Name"),
                      TH("Site"),
                      TH("Start"),
                      TH("Duration/Ending"),
                      TH("Link"),
-                     TH("Add Reminder")))
+                     TH("Add Reminder",
+                        BUTTON(I(_class="fa fa-refresh"),
```

Comment:
Some users may not have Google account, or are only interested in contests list. Loading a Google Calendar authorization on dialog on page load would break UX. 

I may check on load whether the user has already authorized StopStalk so that the dialog would instantly disappear.